tests/test_random.cpp

```diff
@@ -0,0 +1,53 @@
+#include "utils.h"
+#include <catch2/catch.hpp>
+
+#include "faasm/random.h"
+
+using namespace faasm;
+
+namespace tests {
+TEST_CASE("Test random integer", "[random]")
+{
```

Comment:
Is it deliberate that we don't clean anymore? (See previous comment)

Reply:
Yep, this is only using random number generation so doesn't require cleaning any stateful parts of the system.